docroot/modules/custom/sitenow_dispatch/src/Form/SettingsForm.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace Drupal\sitenow_dispatch\Form;
+
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Configure sitenow_dispatch settings for this site.
+ */
+class SettingsForm extends ConfigFormBase {
+
+  /**
+   * The config factory service.
+   *
+   * @var Dispatch
+   */
+  protected $dispatch;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'sitenow_dispatch_settings';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function getEditableConfigNames() {
+    return ['sitenow_dispatch.settings'];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function __construct(ConfigFactoryInterface $config_factory, $dispatch) {
+    parent::__construct($config_factory);
+    $this->dispatch = $dispatch;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container) {
+    return new static(
+      $container->get('config.factory'),
+      $container->get('sitenow_dispatch.dispatch')
+    );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    $form['description_text'] = [
+      '#markup' => '<p>This API key is for the Dispatch service, references are listed below for ease of use.</p><p><a href="https://apps.its.uiowa.edu/dispatch/help">Dispatch documentation</a></p><p><a href="https://apps.its.uiowa.edu/dispatch/api-ref">API reference</a></p>',
+    ];
+    $form['API_key'] = [
+      '#type' => 'textfield',
+      '#title' => $this->t('API key'),
+      '#default_value' => $this->config('sitenow_dispatch.settings')->get('API_key'),
```

Comment:
Nitpick: lower case the config key?

Reply:
you watched me put this in there! you just wanted me to do more work >:(